### DISCUSSION TITLE
image scan error: scan error: scan failed: failed analysis: analyze error: timeout: context deadline exceeded : general response handler: unexpected status code: 500, expected: 200

### DISCUSSION THREAD

**Discussion by hanujawak** [aquasecurity/trivy#3455] [Q&A]
getting **FATAL   image scan error: scan error: scan failed: failed analysis: analyze error: timeout: context deadline exceeded** for trivy CLI as well with Harbor integration.
increased timeout duration from the default 5 min to 20 min. But nothing changed.
image size 2.3GB

**Comment by knqyf263** [comment]
This issue might be relevant.
https://github.com/aquasecurity/trivy/issues/3421
